common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -110,12 +111,18 @@
                     '</div>'
                 ].join('');
```

Comment:
`langTemplate` could be renamed to `langHtml` as well (see note below).  You could also wrap this call in HtmlUtils.HTML() to declare it an HTML Snippet, and then follow the same pattern with `toString()` when adding to the element.

These are somewhat nit-picky changes, but I think this may help us write safer code.  

@andy-armstrong: feel free to weigh in on my comments.